extensions/amp-analytics/0.1/vendors/subsciptions-propensity.js

```diff
@@ -0,0 +1,103 @@
+/**
```

Comment:
typo in the file name. `subscriptions-propensity`. This is why the build is failing.